client/src/components/Layout/Navbar/styles.ts

```diff
@@ -0,0 +1,19 @@
+import styled from "styled-components";
+
+export const Navigation = styled.nav`
```

Comment:
let's keep convention with naming parent styled component like StyledNavbar etc. even if there is no styles